### DISCUSSION TITLE
Can help me to check this for the networkx.hierachy with constraint relevant formulae

### DISCUSSION THREAD

**Discussion by khoo2002** [networkx/networkx#5887] [Q&A]
Hi everyone,

I would like to ask for help to check my function. The function under the structural hole is to count the hierarchy value for a node. The hierarchy is an indicator that measures the extent to which the aggregate constraint on ego is concentrated in a single contact.

The code is here.
https://github.com/khoo2002/Hierarchy_calculation/blob/a452ff6db617854be63b20839e34d4d86ff6bd13/Hierachy_code.ipynb

Please check the link preview at the browser right bottom to ensure you are going to the right link.
Thank you.

**Comment by khoo2002** [comment]
Correcting the typo error
the link is here:
https://github.com/khoo2002/Hierarchy_calculation/blob/41910daef8f36cab8968977da2223897537f449a/Hierarchey_code.ipynb

Please check the link preview at the browser right bottom to ensure you are going to the right link.

**Comment by rossbar** [comment]
Hi @khoo2002 - the networkx discussion forum here is more intended for questions about networkx functions, rather than checking individual user code. Also - more generally - to get feedback on code you should embed it via a code block in markdown, i.e.

````
```python
<your code here>
```
````

as it is generally a best-practice *not* to click on random download links on the internet!

**Reply by khoo2002** [reply]
Ok. I am sorry. There are proof and reference pictures, codes and tables. It is hard for me to embed here. So I choose to upload the jupyter notebook file.  I will try to embed it via a code block in markdown.
